### ISSUE TITLE
Support :where pseudo class

### ISSUE THREAD

**Issue by jannispl** [fb55/css-select#579]
Would it be sensible to implement support for the `:where` pseudo class selector?

https://developer.mozilla.org/en-US/docs/Web/CSS/:where

**Comment by fb55** [comment]
Happy to accept a PR for this! You'd have to add this to `css-what` in

https://github.com/fb55/css-what/blob/1e523da931338953c68cc6e0d7a1e98639aa62c9/src/parse.ts#L107-L114

And then insert the new alias here:

https://github.com/fb55/css-select/blob/30013a5ad58d40c0de4867cca0e8035f2add0d37/src/pseudo-selectors/subselects.ts#L54-L58

Also please add a basic test cases to make sure this will stay available.

**Comment by jannispl** [comment]
Are you suggesting that `:where` should be implemented as an alias for `:is`/`:matches`? I am not entirely certain if this library has to respect specificity, which works differently for `:where`:

>The difference between :where() and :is() is that :where() always has 0 specificity, whereas :is() takes on the specificity of the most specific selector in its arguments.

**Comment by fb55** [comment]
Yes, I am suggesting to just create an alias. Specificity is important for deciding which selectors have precedence for which element when applying styles. That is out of scope for this library (and can be implemented in a separate module if needed).

**Comment by jeripeierSBB** [comment]
Critters is waiting for this fix too: https://github.com/GoogleChromeLabs/critters/issues/90

**Comment by fb55** [comment]
Published as `v4.2.0`.

**Comment by kyubisation** [comment]
Thank you very much 😃
